frontend/src/components/helpers/auth/RequireAdmin.js

```diff
@@ -13,7 +13,7 @@ const RequireAdmin = () => {
   if (!isAdmin) {
     return <Unauthorized />;
   }
-  if (isOpenSource) {
+  if (!isOpenSource) {
```

Comment:
@tahierhussain Why was this changed?

Reply:
@Deepak-Kesavan Thanks for the catch. I had made this change for testing purpose. I have reverted the necessary changes.